### DISCUSSION TITLE
Add delete snippet command

### DISCUSSION THREAD

**Discussion by abloch** [knqyf263/pet#287] [Ideas]
pet has new/edit snippet but doesn't have a simple delete command.

This should be similar to the search command but prompts deleting a snippet instead.

**Comment by RamiAwar** [comment]
I like this idea, I think we can definitely plan this for the future.

Right now, you could use `pet edit` to manually delete a snippet, so this won't get very high priority compared to other feature requests.

**Comment by marilynantonio03-collab** [comment]
> pet has new/edit snippet but doesn't have a simple delete command.
> 
> 
> 
> This should be similar to the search command but prompts deleting a snippet instead.
